src/dependency-manager/src/component-config/v1.ts

```diff
@@ -151,12 +153,15 @@ export class ComponentConfigV1 extends ComponentConfig {
   @Transform((value) => !value ? {} : value)
   interfaces?: Dictionary<InterfaceSpecV1 | string>;
 
-  getName() {
-    return this.name.split(':')[0];
+  //TODO:87:can we add getComponentName(), getAccountName(), getTag()?
```

Comment:
Would you be opposed to adding two methods here: `getAccountName()` and `getComponentName()`?

One of my goals with this is that we're not trying to do a `split('/')` everywhere else across the codebase. But if we only expose the `getName()` method here, that's what we're going to end up doing again.